### ISSUE TITLE
Top 5 Google Developer Console problems from 1.10.1 users, #2

### ISSUE THREAD

**Issue by billmeek** [MythTV-Clients/MythTV-Android-Frontend#181]
Please see #180 for an explanation of the following.

```
latest: Aug 2, 2013, 1 comment: Crashed when syncing with backend
java.lang.NoSuchMethodError: java.lang.String.isEmpty
at org.mythtv.services.api.ETagInfo.isEmptyEtag(ETagInfo.java:64) VERSION 0.0.30

    public boolean isEmptyEtag(){
        return eTag == null || eTag.isEmpty(); <-----------------------------
    }

Same code exists in the 1.0.4 version.

at org.mythtv.services.api.AbstractOperations.getRequestEntity(AbstractOperations.java:114)
at org.mythtv.services.api.dvr.impl.DvrTemplate.getRecordedList(DvrTemplate.java:351)
at org.mythtv.service.dvr.RecordedDownloadService.download(RecordedDownloadService.java:160)
at org.mythtv.service.dvr.RecordedDownloadService.onHandleIntent(RecordedDownloadService.java:127)
at android.app.IntentService$ServiceHandler.handleMessage(IntentService.java:59)
at android.os.Handler.dispatchMessage(Handler.java:99)
at android.os.Looper.loop(Looper.java:144)
at android.os.HandlerThread.run(HandlerThread.java:60)
```

**Comment by dmfrey** [comment]
are there any details on the version of java, phone make & model, etc.

After researching a bit, this looks like something that was running in a java 5 compatible Dalvik VM.  The isEmpty() function in the String class was introduced in java 1.6.

**Comment by billmeek** [comment]
A70S; Android 2.1 per http://www.handsetdetection.com/properties/vendormodel/Archos/A70S

and

Desire (bravo); Android 2.1, per: http://www.gsmchoice.com/en/catalogue/htc/desire/

 were the Devices used.

**Comment by dmfrey** [comment]
closes #181

The modern devices we are supporting going forward should not fall into
this category anymore.
